src/style.css

```diff
@@ -1,3 +1,100 @@
+@font-face {
+  font-family: "Roboto";
+  src: url("fonts/Roboto-Medium.ttf") format("woff2");
+  font-weight: 500;
+  font-style: normal;
+}
+
+html {
+  --WHITE_COLOR: #fff;
+  --BLACK_COLOR: #000;
+  --AQUA_GRADIENT: #00acdc;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  height: 60px;
+  background-color: var(--WHITE_COLOR);
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+  width: 100%;
+  box-sizing: border-box;
+}
+
+.logo-container {
+  display: flex;
+  flex-shrink: 3;
+  flex-grow: 1;
+}
+
+.navigation-block {
+  flex-shrink: 0;
+  flex-grow: 3;
+  display: flex;
+  justify-content: flex-end;
+}
+
+.logo-container a {
+  height: 40px;
+  width: 40px;
+  align-self: center;
+}
+
+.navigation > .menu-nav {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  height: 60px;
+  font-size: 12px;
+}
+
+.menu-nav li {
+  text-transform: uppercase;
+  display: flex;
+  align-items: center;
+  height: 100%;
+  position: relative;
+}
```

Comment:
please set the classname for tag li, and set styles by class name instead of tags